.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+name: Format Checker
+
+on:
+    push:
+        branches: [main]
+    pull_request:
+        branches: [main]
```

Comment:
Shouldn't this preferably run on all pull requests, and not only the main branch? Or does this mean it'll only run on PRs that targets the main branch?

Reply:
Yeah god idea i fix this now

Reply:
Done